Dockerfile.ubuntu

```diff
@@ -43,11 +44,18 @@ RUN apt-get install -y cmake libboost-python-dev python-dev
 # ZBar
 RUN apt-get install -y libzbar-dev python-dev
 
+# pip (need a recent version e.g. for wheels support - ubuntu is obsolete)
+RUN wget https://bootstrap.pypa.io/get-pip.py
+RUN python get-pip.py
```

Comment:
Using apt-get first is slightly more secure, but it also allows the upgrade
to be conditional on the apt version of pip  being too low for the feature
desired.

On Tue, 12 Jul 2016 00:38 drtrigon, <notifications@github.com> wrote:

> In Dockerfile.ubuntu
> <https://github.com/AbdealiJK/file-metadata/pull/48#discussion_r70302367>:
>
> > @@ -43,11 +44,18 @@ RUN apt-get install -y cmake libboost-python-dev python-dev
> >  # ZBar
> >  RUN apt-get install -y libzbar-dev python-dev
> >
> > +# pip (need a recent version e.g. for wheels support - ubuntu is obsolete)
> > +RUN wget https://bootstrap.pypa.io/get-pip.py
> > +RUN python get-pip.py
>
> # Pip - install latest version so that wheels are used
> RUN apt-get install -y python-pip && \
>   pip install -U pip
>
> works, but gives:
>
> Installing collected packages: pip
>   Found existing installation: pip 1.5.4
>     Not uninstalling pip at /usr/lib/python2.7/dist-packages, owned by OS
> Successfully installed pip
>
> which gives me the impression that now 2 different pips are installed. I
> don't like that.
>
> What is reasoning behind your comment? You would like to avoid the wget
> of a mysterious file that gets installed into the system then? Security
> issues?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/AbdealiJK/file-metadata/pull/48/files/f93f3073c7aa0cffef3ddd8830118111b48a31da#r70302367>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAA69I58Oz8ldx007VEUXD-LTBZ1Py2Qks5qUn-wgaJpZM4JI7QH>
> .
>